server/main.go

```diff
@@ -13,7 +13,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/privacypass/challenge-bypass-server"
+	btd "github.com/privacypass/challenge-bypass-server"
```

Comment:
What is this for? It built for me with Go 1.13.5 without this change.

Reply:
Oh, it must have been my lint adding things. I will remove ;)

Reply:
Could you squash this change in the previous change? Thanks :)

Reply:
Done ;)